recipes/libpng/all/conanfile.py

```diff
@@ -2,7 +2,7 @@
 from conans import CMake, tools
 from conan import ConanFile
 from conan.tools.scm import Version
-from conan.tools.files import apply_conandata_patches, get, rmdir, replace_in_file
+from conan.tools.files import apply_conandata_patches, get, rm, rmdir, replace_in_file
 from conan.tools.build.cross_building import cross_building
 
 required_conan_version = ">=1.50.2"
```

Comment:
```suggestion
required_conan_version = ">=1.50.2 <1.51.0 || >=1.51.2"
```

To related to this PR direcrlt, but here is a fix to `required_conan_version` (due to usage of major/minor in conan.tools.scm.Version).